app/controllers/application_controller.rb

```diff
@@ -79,7 +79,7 @@ def requested_locale
   end
 
   # Around filter to ensure locale (language and timezone) are set only per request
-  def set_locale
+  def set_locale(&action)
```

Comment:
The Rails guides have been updated to account for the possibility of locale setting leaking across requests. This change updates `set_locale` to match the suggested method signature.